spec/command_recorder_spec.rb

```diff
@@ -37,7 +37,7 @@ def before(env)
     end
 
     it "does not fail when reverting" do
-      migration = Class.new ::ActiveRecord::Migration do
+      migration = Class.new ::ActiveRecord::Migration[5.0] do
```

Comment:
this allows the spec to pass, but is probably not the right solution:

`env.options[:index] = { name: 'whatever' } if enabled_middleware(AddIndexBefore, env)`

(in this case, the call to `enabled_middleware` always returns false)

I need to better understand what the desired behaviour should be.